### DISCUSSION TITLE
nwg-look 0.2.7 (2024-01012)

### DISCUSSION THREAD

**Discussion by nwg-piotr** [nwg-piotr/nwg-shell#289] [Changelog]
https://github.com/nwg-piotr/nwg-look/releases/tag/v0.2.7

**Comment by d2718** [comment]
I just want to say that I just now stumbled upon `nwg-look`, and it was _exactly_ what I needed. Awesome; thank you.

<3

**Reply by nwg-piotr** [reply]
Thank you!
